js_modules/dagit/packages/core/src/assets/__fixtures__/AssetsCatalogTable.mocks.ts

```diff
@@ -355,6 +358,7 @@ export const AssetCatalogTableMock: MockedResponse<AssetCatalogTableQuery> = {
               id: 'test.py.repo.["good_asset"]',
               groupName: 'GROUP2',
               isSource: false,
+              isObservable: false,
```

Comment:
Not sure if you need this for your tests or if its just for typescript but If its the latter then using the `build` functions will make it so you don't need to update the mocks again in the future.

Reply:
Ahh let me switch these out to the build functions that would be nice here actually!